src/site/content/en/lighthouse-pwa/viewport/index.md

```diff
@@ -1,57 +1,66 @@
 description: |
-  Learn about `viewport` audit.
+  Learn about the "Does not have a <meta name="viewport"> tag with width or
+  initial-scale" Lighthouse audit.
+updated: 2019-08-20
 web_lighthouse:
   - viewport
 ---
 
-Without a viewport meta tag, mobile devices render pages at typical desktop
-screen widths, and then scale the pages to fit the mobile screens.
-Lighthouse flags pages without a viewport:
+Many search engines rank pages based on how mobile-friendly they are.
+Without a [viewport meta tag](https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag),
+mobile devices render pages at typical desktop screen widths and then scale the
+pages down, making them difficult to read.
+
+Setting the viewport meta tag lets you control the
+width and scaling of the viewport so that it's sized correctly on all devices.
+
+## How the Lighthouse viewport meta tag audit fails
+
+Lighthouse flags pages without a viewport meta tag:
 
 <figure class="w-figure">
-  <img class="w-screenshot w-screenshot--filled" src="viewport.png" alt="Lighthouse audit shows page is missing a viewport">
-  <figcaption class="w-figcaption">
-    Page is missing a viewport.
-  </figcaption>
+  <img class="w-screenshot w-screenshot" src="viewport.png" alt="Lighthouse audit shows page is missing a viewport">
 </figure>
 
-## How this audit fails
+A page fails the audit unless all of these conditions are met:
+- The document's `<head>` contains a `<meta name="viewport">` tag.
+- The viewport meta tag contains a `content` attribute.
+- The `content` attribute's value includes the text `width=`.
 
-Lighthouse checks that there's a `<meta name="viewport">` tag in the `<head>`
-of the document. It also checks that the node contains a `content` attribute
-and that the value of this attribute contains the text `width=`. However,
-it does not check that `width` equals `device-width`. Lighthouse also does not
-check for a `initial-scale` key-value pair.
+Lighthouse _doesn't_ check that `width` equals `device-width`. It also doesn't
+check for an `initial-scale` key-value pair. However, you still need to include
+both for your page to render correctly on mobile devices.
 
 {% include 'content/lighthouse-pwa/scoring.njk' %}
 
-## Recommendations
+## How to add a viewport meta tag
 
-Add a viewport `<meta>` tag in the `<head>` of your HTML:
+Add a viewport `<meta>` tag with the appropriate key-value pairs to the `<head>`
+of your page:
 
-```html
-<head>
-  ...
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  ...
-</head>
+```html/4
+<!doctype html>
+<html lang="en">
+  <head>
+    …
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    …
+  </head>
+  <body>
+    …
+  </body>
+</html>
 ```
 
-The `width=device-width` key-value pair sets the width of the viewport to
-the width of the device. The `initial-scale=1` key-value pair sets the initial
-zoom level when visiting the page.
-
-## Why you should include a viewport
```

Comment:
I saw this a couple days ago and it bugged the hell out of me. We're not adding a viewport, we're adding a viewport meta tag...